packages/language-server/lib/server.js

```diff
@@ -22,7 +23,22 @@ connection.onInitialize(() => {
 documents.onDidChangeContent(async (event) => {
   const diagnostics = await validateDocument(event.document);
   connection.sendDiagnostics({ uri: event.document.uri, diagnostics });
+
+  if (diagnostics.length > 0) {
+    // The language server (currently) only exposes **syntactic** XML diagnostics.
+    getSWA().track("Syntax Issues Detected", [{ number: diagnostics.length }]);
```

Comment:
I think the second parameter should be an array of strings

Reply:
I see, I will consult with @asafdl  what can be passed here.
And inspect the tracking results with @allacherkes .

I wonder if I should stringify a JSON or just pass a number...

Reply:
Please check if you can pass a number or need to stringify it (since the method expects a string array)

Reply:
Checked successfully on canary